docs/_Aug_milestone_.md

```diff
@@ -0,0 +1,24 @@
+## August Milestone
+
+### Overview
+
+- Finish hasManyThrough relation MVP
+- Continue with docs improvements
+
+### Goals
+
+- [ ] hasManyThrough relation MVP
+
+  - [ ] docs: add HasManyThrough relation #5835
+  - [ ] feat: set up HasManyThrough relation through the CLI #5836
+
+- [ ] Documentation improvement
+
+  - [ ] Rename "legacy juggler bridge" mentioned in docs #5558
+  - [ ] Add Couchdb connector docs #5267
+  - [ ] [APIDocs] Table not showing properly when it includes code snippet #5756
+
+- [ ] PRs to review:
+  - [ ] any PRs?
+
+### Stretch Goals
```

Comment:
Let's add:

- Blogpost: what are our current plans for juggler modernization #5956